src/System.Net.HttpListener/tests/HttpListenerRequestTests.cs

```diff
@@ -143,25 +143,6 @@ public async Task ContentLength_GetProperty_ReturnsExpected(string method, strin
             Assert.Equal(hasEntityBody, request.HasEntityBody);
         }
 
-        [Theory]
-        [InlineData(100)]
-        [InlineData("-100")]
-        [InlineData("")]
-        [InlineData("abc")]
-        [InlineData("9223372036854775808")]
-        [ActiveIssue(20294, TargetFrameworkMonikers.Netcoreapp)]
-        public async Task ContentLength_ManuallySetInHeaders_ReturnsExpected(string newValue)
-        {
-            HttpListenerRequest request = await GetRequest("POST", null, new string[] { "Content-Length: 1" }, content: "\r\n");
-            Assert.Equal("1", request.Headers["Content-Length"]);
-
-            request.Headers.Set("Content-Length", newValue);
-            Assert.Equal(newValue, request.Headers["Content-Length"]);
-            Assert.Equal(1, request.ContentLength64);
-
-            Assert.True(request.HasEntityBody);
-        }
-
```

Comment:
I deleted this because the associated ActiveIssue was for a closed issue saying that this test was bogus.